packages/pyright-internal/src/common/stringUtils.ts

```diff
@@ -7,51 +7,25 @@
  * Utility methods for manipulating and comparing strings.
  */
 
-import leven from 'leven';
-
 import { compareComparableValues, Comparison } from './core';
 
-// Determines how closely a typed string matches a symbol
-// name. An exact match returns 1. A match that differs
-// only in case returns a slightly lower number. A match
-// that involves a few missing or added characters returns
-// an even lower number.
+// Determines if typed string matches a symbol
+// name. Characters must appear in order.
+// Return 1 if all typed characters are in symbol
 export function computeCompletionSimilarity(typedValue: string, symbolName: string): number {
```

Comment:
can you leave this code here and add another method that uses isPatternWord? asking since I still need old one for code action.

Reply:
Yes, the old one should stay (and its dependency left in package.json). We can remove the dead code later if we eliminate it.